docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.masonry.scss

```diff
@@ -0,0 +1,65 @@
+// citing use of this codepen https://codepen.io/chriscoyier/pen/NeRNBO
+
+// SOLN 1 - CSS ONLY
```

Comment:
this is soln 1. comment out the masonry JS to see this in action and the CSS below